drf_rw_serializers/generics.py

```diff
@@ -52,7 +36,8 @@ def get_serializer_class(self):
                     "attribute, or override the `get_read_serializer_class()` or "
                     "`get_serializer_class()` method." % self.__class__.__name__
                 )
-                return self.get_read_serializer_class()
+                # `use_serializer_class` is used to prevent a `RecursionError`
+                return self.get_read_serializer_class(use_serializer_class=True)
```

Comment:
Let me know if you think another name (`use_serializer_class`) is better.

Reply:
use_serializer_class -> default_to_serializer_class perhaps?

Reply:
Done, thanks